web/libs/editor/src/components/Comments/Comments.tsx

```diff
@@ -20,7 +20,11 @@ export const Comments: FC<{ annotationStore: any; commentStore: any; cacheKey?:
     const mounted = useMounted();
 
     const loadComments = async () => {
-      await commentStore.listComments({ mounted });
+      const listCommentsOptions: any = { mounted };
+      if (isPerFieldComments) {
+        listCommentsOptions.suppressClearComments = commentStore.isRelevantList;
+      }
```

Comment:
It allows not to reset comment list on each comments tab activation. It prevents blinking.